### DISCUSSION TITLE
ESOH Model errors

### DISCUSSION THREAD

**Discussion by sklein-ise** [pybamm-team/PyBaMM#1828] [Q&A]
Hey,

I've been getting a lot of errors, when I try to use the experiment class without calc_esoh = False and I don't really understand how to fix it.

The idea was to cycle a battery between 75% to 50% SoC for approximately 2400 cycles with ec reaction limited SEI growth.
I wanted to fit the "SEI kinetic rate constant [m.s-1]" to the experiment by simulating for different kinetic rates between 10**(-14) and 10**(-18).

When it tries to solve for the ESOH model I get
1) `SolverError: Could not find acceptable solution: .../casadi/core/rootfinder.cpp:280: rootfinder process failed. Set 'error_on_fail' option to false to ignore this error.`

2) or an error where it says the found solution error is higher than the tolerance

What do these errors mean and is there any way to avoid this?

Thanks in advance

**Comment by valentinsulzer** [comment]
It means it wasn't able to solve the esoh model. Can you share the code?
